packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -718,4 +718,21 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
   private _noLegendHighlighted = () => {
     return this.state.selectedLegend === '' && this.state.activeLegend === '';
   };
+
+  private _addDefaultColors = (lineChartData?: ILineChartPoints[]): ILineChartPoints[] => {
+    return lineChartData
+      ? lineChartData.map((item, index) => {
+          let color: string;
+          if (typeof item.color === 'undefined') {
+            color = getNextColor(index, 0, this.props.theme?.isInverted);
+          } else if (!item.color.startsWith('#')) {
+            color = getColorFromToken(item.color, this.props.theme?.isInverted);
```

Comment:
move this logic of # from here to getColorFromToken.
having # is not a guarantee. User could be using rgb(255, 255, 255) or primary colors like red, green, blue etc.

Reply:
fixed